packages/webapp/pages/squads/create.tsx

```diff
@@ -121,6 +121,7 @@ function CreatePost(): ReactElement {
     onSuccess: (newSquad) => {
       const form = formToJson<CreatePostProps>(formRef.current);
 
+      setSelected(selected + 1);
```

Comment:
A minor hack for aesthetic purposes.

Moving the "create new squad" option to the bottom created some funky UI behavior. The navigation often takes a second before it happens after the squad is created. This results in the wrong squad being shown as selected for a brief moment as the list grows, but the selected index remains the same:


https://github.com/user-attachments/assets/ce595de2-49b6-41cb-b93c-27acc4f89bdd



There's certainly a root problem that should/could be addressed here, but I think this is an OK quick fix for now.

Reply:
😬

Reply:
I would actually avoid the quick fix and this fix in this case then have something not really stable for solving the issue.  🤔 
Also what's also annoying me that I hadn't noticed before is the flickering in the dropdown when u select a value

Reply:
@nensidosari The potentially wonky incident I can see is if the squad gets created, but post doesn't. In that case if the user clicks post again, he will create a second squad. But yeah, if you want to avoid the hack, or let this issue rest all together for now it's understandable.

The crux of the issue here is that the dropdown uses indexes and not unique identifiers, but refactoring that seemed like way out of the scope of this task. Also I've seen talk about potentially adopting radix in the future, which would make fixing the dropdown redundant.


> Also what's also annoying me that I hadn't noticed before is the flickering in the dropdown when u select a value

Yeah, seems to be caused by multiple rerenders. Quite unsightly 😅

Reply:
> Also what's also annoying me that I hadn't noticed before is the flickering in the dropdown when u select a value

That's an on-going issue with our context menu 😅

Reply:
@AmarTrebinjac the problem we're fixing here, does it mean we fetch squads again in the dropdown now that a new squad is added that causes the weird behaviour?

Reply:
@AmarTrebinjac I think we can just remove this line. After creation, I think it won't matter whether the label from "Create new Squad" would become "User's private Squad". The intention was to create a new Squad and the label itself displays the accurate value.